backend/createInterProEntries.sh.in

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Parses the interpro entries
+
+set -o pipefail -e
+exec 3>&1
+
+entries="ftp://ftp.ebi.ac.uk/pub/databases/interpro/entry.list"
```

Comment:
is this available from a non-ftp source? We usually run the database creation on a high-memory node of the HPC and ftp is blocked there.

Reply:
It seems to work with the same url, but starting with `http://`
```suggestion
entries="http://ftp.ebi.ac.uk/pub/databases/interpro/entry.list"
```

Reply:
Changed in pcvthien/unipept@07c941a60b9ed22acdafb61e7ab5d1f62e5621a3